src/OrchardCore/OrchardCore.Navigation.Core/NavigationManager.cs

```diff
@@ -227,12 +227,16 @@ private async Task<List<MenuItem>> AuthorizeAsync(IEnumerable<MenuItem> items, C
                 }
                 else
                 {
+                    //evaluate permissions in AND way
```

Comment:
I don't totally understand this change. Can you expand the comment?

Reply:
Because the Permission selector is a multi-permission selector. Maybe it should be a single permission selector only.

Reply:
Ah, I see.

I think it should probably be a single selector (that's worked perfectly well up to now? I think?)

Unless @PiemP you can find me an example where a menu should be displayed based on two different permissions?

Reply:
Honestly I don't have an example. If someone create a module that do something strange over a the items of a specific content type and the user want to show the menu item only if the user can publish and see all the content item of this content type...I don't know...Probably it's better to ask it to the guy that have create the `MenuItem` object with a structure for the property `Permissions`. In this moment I don't have this needs in my site.

For my opinion the only issue about that is about the evaluation of multiple permissions: that is done only in "AND way" and the user can't choose to change this behavior. We can only loose a possibility, if we remove this code. And ff we limit the permission picker to select only one permission? In the mean time we can think about what do with this.  

Thank you guys, I hope to be explained me well. 

PS. this code fix an issue. Before, if someone add plus than one permission to a MenuItem, this menu item was show multiple time.

Reply:
Let's keep it and see then. I totally agree that having it is better than not having it. We will see over time how people will use it.